src/Sylius/Bundle/ApiBundle/Exception/TaxonCannotBeRemoved.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Exception;
+
+/** @experimental */
+final class TaxonCannotBeRemoved extends \RuntimeException
+{
+    public function __construct(string $message = 'Cannot delete, the taxon is in use.')
```

Comment:
If this PR will be merged first https://github.com/Sylius/Sylius/pull/15322/files
please synchronize updated constructor signature.

Reply:
#15322 is merged :v